js/sound.js

```diff
@@ -198,3 +198,13 @@ function playAudio(audio, licenseNum) {
     audioPlay.play();
     $("#tileLicense").html(creativeCommonsLicense[licenseNum - 1]);
 }
+
+function playPauseFloat() {
+    if(backgroundPlay.paused === true) {
+        startBackgroundAudio();
+        document.getElementById("play-pause-float").innerHTML = "<i class=\"fa fa-pause play-pause-float-i\"></i>"; 
```

Comment:
You can also modify the class of the "i" element. This way, we do not mix JS and HTML.

Reply:
Then how am I supposed to change the icon?

Reply:
So, it now only changes the class of the element rather than changing it entirely. I think this fixes the problem!